huggingface/pytorch/tgi/docker/1.0.2/py3/sdk2.15.0/Dockerfile

```diff
@@ -0,0 +1,140 @@
+# Fetch and extract the TGI sources
```

Comment:
Instead of including the sources for the tgi server in this repo, I would rather use those from the optimum-neuron repo that are likely to be up-to-date.

Reply:
You can cherry-pick this commit from my fork for instance:
https://github.com/dacorvo/llm-hosting-container/commit/17730ff5b6112b2f0aee8515a5f3c03545ec70f3

Reply:
If you look at my branch I keep the Dockerfile, but I added a step to fetch the optimum-neuron sources and install from them directly.

Reply:
ah nice can you open a PR to my branch then we can keep this one?

Reply:
Note that I updated my branch to also remove the server files, so the commit i referenced before is invalid.
You can still `git cherry-pick https://github.com/dacorvo/llm-hosting-container/tree/neuronx-tgi` to get the correct commit.

Reply:
we can also close this PR and open a new one.

Reply:
updated 👍🏻